### PR TITLE
#22 Fix: Header margin top on mobile

### PR DESCRIPTION
**Pull request by MugaboDenys**
### Task Description _(❗️Required)_
I fixed margin top on mobile screens to 27px(in rems)
### Type of Task

- [x] fix
- [ ] chore
- [ ] feat
- [ ] refactor
- [ ] perf
- [ ] docs

### Guidelines

- [x] Does your code follow ES6+ syntax?
- [x] Did you use arrow functional components?
- [x] Did you thoroughly review your code?
- [x] Have you checked if your code does not introduce conflicts in the main codebase?

### Ticket Link _(❗️Required)_

https://www.notion.so/apeunit/header-section-mobile-top-margin-4649cd50b0714657b8d31fc28a0f8da6

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -72,6 +72,7 @@ module.exports = {
         4.5: "1.1875rem",
         5.1: "1.3125rem",
         6.25: "1.5625rem",
+        6.5 : "1.6875rem",
```
Comment (mercihabam) [inline tailwind.config.js]: To get the value to use as tailwind key we take the value in pixels divide by 4. for your case is `` 27px / 4 = 6.75``

**Review by mercihabam — CHANGES_REQUESTED** [review]
## You've made it.

Just on small thing to change, kindly refer to my comment.

File: tailwind.config.js (hunk 1 above)
Comment (MugaboDenys) [reply tailwind.config.js]: Thank you @jacob-js

**Review by Odile078 — APPROVED** [review]

**Review by alain-kubwayo — APPROVED** [review]

**Review by Lily-Lilian — APPROVED** [review]
LGTM!

**Review by clever-cr — APPROVED** [review]

**Review by mercihabam — APPROVED** [review]

**Pull request merged**